data/brands/shop/hairdresser.json

```
@@ -109,6 +109,19 @@
         "shop": "hairdresser"
       }
     },
+    {
+      "displayName": "Cutters",
+      "id": "cutters-a2b284",
+      "locationSet": {"include": ["no"]},
```

Based on their [website](https://www.cutters.no/) it looks like they also have locations in Finland and Sweden
```suggestion
      "locationSet": {"include": ["fi", "no", "se"]},
```

---

Yes, you are right. I read through the Norwegian Wikipedia article only, which start of with "Cutters are a Norwegian hairdresser chain..."